docs/examples/robust_phase_estimation.ipynb

```diff
@@ -705,4 +705,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
+}
```

Comment:
No newline at EOF- would normally be a flake8 violation (is flake8 used in this repo?).

Reply:
Does flake8 check notebooks by default? Thought you needed some special addon for that.